### ISSUE TITLE
Subtitles not downloading

### ISSUE THREAD

**Issue by CoreInfini7y** [oop7/YTSage#1]
I have tried to download english subtitles for this: https://youtu.be/CzHJK4Qsrow?si=9Hr0wrNs0UiiaTJw
First click on the "Download subtitles" revealed the list of available manual and auto-generated subtitles, choosing "en - Manual" and clicking again on the button doesn't do a download just hides the subtitles list.

Debian 12, Python 3.11.2

If you need any details, please do not hesitate.

**Comment by oop7** [comment]
Could you please give me the log?

**Comment by CoreInfini7y** [comment]
Where can I find it? There is no logging in the source code and also no log dir or file in YTSage subdir.

**Comment by oop7** [comment]
Run the .py file, and you will get the log in your terminal.

**Comment by CoreInfini7y** [comment]
Thanks. So, I discovered a user error as I re-downloaded the same video and before doing that, I also set the subtitle for download and it arrived together with the video.

No issue, however, I would recommend to make it obvious that subtitles can be downloaded together with the video and it's not possible after the video is already down. I also recommend to convert the subtitle to srt or offer an option for this on the GUI.

Otherwise, keep up the good work, it's really a program that worth every second you spent on the development!
